src/App.js

```diff
@@ -85,19 +86,35 @@ export default class App extends Component {
   }
 
   resetDogs = (event) => {
-    console.log('adoptable', this.state.adoptableDogs)
     let dogs = this.allDogs(this.state.adoptableDogs);
     this.setState({
       filteredDogs: dogs
     })
   }
 
+  filterBreed = (event) => {
+    this.state.selectedBreed = [];
+    let breedNodes = document.querySelectorAll('#breed');
+    for (let i = 0; i< breedNodes.length; i++) {
+      if (breedNodes[i].selected === true) {
+      this.state.selectedBreed.push(breedNodes[i].value)
+      console.log(this.state.selectedBreed)
+      console.log(breedNodes[i].value)
+      }
+    }
+    let dogzzzz = this.allDogs(this.state.adoptableDogs)
```

Comment:
Make sure you update this variable name to be a bit more descriptive of what is being held under this label.